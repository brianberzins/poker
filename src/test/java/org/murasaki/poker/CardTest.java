package org.murasaki.poker;

import org.approvaltests.combinations.CombinationApprovals;
import org.junit.jupiter.api.Test;

class CardTest {

    @Test
    void testCards() {
        CombinationApprovals.verifyAllCombinations(Card::new, Rank.values(), Suit.values());
    }

}
