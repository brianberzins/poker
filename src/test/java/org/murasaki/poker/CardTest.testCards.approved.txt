[DEUCE, HEARTS] => DEUCE of HEARTS 
[DEUCE, SPADES] => DEUCE of SPADES 
[DEUCE, CLUBS] => DEUCE of CLUBS 
[DEUCE, DIAMONDS] => DEUCE of DIAMONDS 
[THREE, HEARTS] => THREE of HEARTS 
[THREE, SPADES] => THREE of SPADES 
[THREE, CLUBS] => THREE of CLUBS 
[THREE, DIAMONDS] => THREE of DIAMONDS 
[FOUR, HEARTS] => FOUR of HEARTS 
[FOUR, SPADES] => FOUR of SPADES 
[FOUR, CLUBS] => FOUR of CLUBS 
[FOUR, DIAMONDS] => FOUR of DIAMONDS 
[FIVE, HEARTS] => FIVE of HEARTS 
[FIVE, SPADES] => FIVE of SPADES 
[FIVE, CLUBS] => FIVE of CLUBS 
[FIVE, DIAMONDS] => FIVE of DIAMONDS 
[SIX, HEARTS] => SIX of HEARTS 
[SIX, SPADES] => SIX of SPADES 
[SIX, CLUBS] => SIX of CLUBS 
[SIX, DIAMONDS] => SIX of DIAMONDS 
[SEVEN, HEARTS] => SEVEN of HEARTS 
[SEVEN, SPADES] => SEVEN of SPADES 
[SEVEN, CLUBS] => SEVEN of CLUBS 
[SEVEN, DIAMONDS] => SEVEN of DIAMONDS 
[EIGHT, HEARTS] => EIGHT of HEARTS 
[EIGHT, SPADES] => EIGHT of SPADES 
[EIGHT, CLUBS] => EIGHT of CLUBS 
[EIGHT, DIAMONDS] => EIGHT of DIAMONDS 
[NINE, HEARTS] => NINE of HEARTS 
[NINE, SPADES] => NINE of SPADES 
[NINE, CLUBS] => NINE of CLUBS 
[NINE, DIAMONDS] => NINE of DIAMONDS 
[TEN, HEARTS] => TEN of HEARTS 
[TEN, SPADES] => TEN of SPADES 
[TEN, CLUBS] => TEN of CLUBS 
[TEN, DIAMONDS] => TEN of DIAMONDS 
[JACK, HEARTS] => JACK of HEARTS 
[JACK, SPADES] => JACK of SPADES 
[JACK, CLUBS] => JACK of CLUBS 
[JACK, DIAMONDS] => JACK of DIAMONDS 
[QUEEN, HEARTS] => QUEEN of HEARTS 
[QUEEN, SPADES] => QUEEN of SPADES 
[QUEEN, CLUBS] => QUEEN of CLUBS 
[QUEEN, DIAMONDS] => QUEEN of DIAMONDS 
[KING, HEARTS] => KING of HEARTS 
[KING, SPADES] => KING of SPADES 
[KING, CLUBS] => KING of CLUBS 
[KING, DIAMONDS] => KING of DIAMONDS 
[ACE, HEARTS] => ACE of HEARTS 
[ACE, SPADES] => ACE of SPADES 
[ACE, CLUBS] => ACE of CLUBS 
[ACE, DIAMONDS] => ACE of DIAMONDS 
